RecoHI/HiTracking/plugins/HIPixelTracksSlimmer_miniAOD.cc

```diff
@@ -0,0 +1,256 @@
+//Based on: PhysicsTools/PatAlgos/plugins/PATLostTracks.cc
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/RecoCandidate/interface/RecoChargedCandidate.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include "DataFormats/Common/interface/Association.h"
+
+#include <vector>
+
+//
+// class declaration
+//
+
+class HIPixelTracksSlimmer_miniAOD : public edm::stream::EDProducer<> {
```

Comment:
I do not see much heavy-ion specific code here.
This seems generic enough to be in the PhysicsTools/PatAlgos/plugins/
after a rename to smth like ```PATTracksToPackedCandidates``` (with or without ```Slimmer``` at the end)

Reply:
Okay, I used a more generic name. This comment is included in this commit: https://github.com/cms-sw/cmssw/pull/30313/commits/deebf6766c730a8c97ac3f3fe90b065b24d5e70d